### PR TITLE
Getting started gizmos notice

### PR DESCRIPTION
**Pull request by RogPodge**
## Overview
Adds helpful notice in the tutorial to lower the gizmos size for the HandInteractionExamples.unity scene
Gizmos

Duplicate of this misconfigured PR: https://github.com/microsoft/MixedRealityToolkit-Unity/pull/7755

### CODE REVIEW

File: Documentation/Contributing/DocumentationGuide.md [hunk 1]
```diff
@@ -158,6 +158,7 @@ When adding a link, consider whether it should also be listed in the [See also](
 
 Some other aspects:
 
+* The editor UI for screenshots should use light gray theme editor as not all users have access to the dark them and we'd like to keep things as consistent as possible.
```
Comment (keveleigh) [inline Documentation/Contributing/DocumentationGuide.md:161]:
```suggestion
* The editor UI for screenshots should use light gray theme editor as not all users have access to the dark theme and we'd like to keep things as consistent as possible.
```

File: Documentation/GettingStartedWithTheMRTK.md [hunk 2]
```diff
@@ -146,6 +146,10 @@ To try the hand interaction scene, do the following steps.
 
 1. Close the TMP dialog. After this you need to reload the scene. You can do this by double-clicking the scene in the Project tab.
 
+1. Uncheck or shrink the size of the 3d icons under the Gizmos tab in the Scene view to make the scene easier to look at
```
Comment (wiwei) [inline Documentation/GettingStartedWithTheMRTK.md:149]:
```suggestion
1. Uncheck or shrink the size of the 3d icons under the Gizmos tab in the Scene view to reduce scene clutter
```

**Comment by keveleigh** [comment]
>It seems like there is unrelated meta file change?

Ah, looks like this branch was originally based on the dev branch instead of the stabilization branch

**Comment by keveleigh** [comment]
>Removing change for stabilization branch

I think we may want to reset this branch on top of the stabilization branch instead of manually removing that file in a different commit. I fear that merging that commit back into mrtk_dev might remove the file? Though I guess I'm not positive on that.

Though, from the command line we can reset this PR via:

```
git reset --hard upstream/prerelease/2.4.0_stabilization
git cherry-pick d93f6999e7d652bd54066320c9079f62a4e9aae0
git cherry-pick 
git cherry-pick 
git cherry-pick 
git cherry-pick 
```

**Comment by RogPodge** [comment]
I ran the commands and I think its good for 2.4

**Pull request merged**